src/testUtils/java/org/broadinstitute/hellbender/testutils/CopyNumberTestUtils.java

```diff
@@ -0,0 +1,76 @@
+package org.broadinstitute.hellbender.testutils;
+
+import htsjdk.samtools.SAMException;
+import htsjdk.samtools.util.IOUtil;
+import org.apache.logging.log4j.Logger;
+import org.testng.Assert;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public final class CopyNumberTestUtils {
+
+    /**
+     * This test utility method asserts if two TSV files (each containing a header followed by a matrix of doubles)
+     * are approximately equivalent, by checking if corresponding double values are equal up to some number of decimal
+     * points represented by the delta parameter. Note that the headers of both files need to be bitwise identical,
+     * and the number of columns and rows of each matrix encoded in TSV files must be equal as well.
+     *
+     * @param f1 first file to compare
+     * @param f2 second file to compare
+     * @param delta the absolute tolerable difference between the actual and the expected values
+     * @param logger the Logger object to use
+     */
+    public static void assertFilesEqualUpToAllowedDeltaForDoubleValues(final File f1, final File f2, final double delta,
+                                                                       final Logger logger) {
+        try {
+            IOUtil.assertFilesEqual(f1, f2);
```

Comment:
I wouldn't bother with this - the parsing code will be super fast for test files anyway.